app/code/core/Mage/Adminhtml/Block/System/Config/Form/Field/Label.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ * @copyright  Copyright (c) 2023 The OpenMage Contributors (https://www.openmage.org)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Adminhtml system config label field renderer
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ */
+class Mage_Adminhtml_Block_System_Config_Form_Field_Label extends Mage_Adminhtml_Block_System_Config_Form_Field
+{
+    public function render(Varien_Data_Form_Element_Abstract $element)
+    {
+        $element->unsScope()->unsCanUseWebsiteValue()->unsCanUseDefaultValue()->unsPath();
+        return parent::render($element);
+    }
+
+    protected function _getElementHtml(Varien_Data_Form_Element_Abstract $element)
+    {
+        $field = $element->getFieldConfig();
+
+        $label = Mage::getModel('Varien_Data_Form_Element_Label');
+        $label->setValue($field->value);
+        $label->setBold(!empty($field->bold));
+
+        return $label->getElementHtml();
```

Comment:
Simplify version:
```suggestion
    /**
     * @inheritdoc
     */
    public function render(Varien_Data_Form_Element_Abstract $element)
    {
        $element->unsScope()->unsCanUseWebsiteValue()->unsCanUseDefaultValue()->unsPath();
        $field = $element->getFieldConfig();
        $element->setValue($field->value);
        $element->setBold(!empty($field->bold));
        
        return parent::render($element);
```

Reply:
Yes, but it's not the way of `Mage_Adminhtml_Block_Customer_System_Config_Validatevat` and `Mage_Adminhtml_Block_System_Config_System_Storage_Media_Synchronize `, so it's probably not the good way to simplify?

But perhaps we can add phpdoc.